tests/login.e2e.ts

```diff
@@ -36,18 +41,18 @@ test('Successful login for teacher', async (t) => {
 });
 test('Failed login for student', async (t) => {
   await loginPage.loginAsStudent();
-  await assertLoginFailure(t);
+  await assertOnLoginPage(t);
 });
 
 test('Failed login - wrong credentials', async (t) => {
   // wrong password
-  await loginPage.login('teacher', 'wrong-password');
+  await loginPage.login(teacherUser, 'wrong-password');
```

Comment:
consider using `Role` factory for this:
https://devexpress.github.io/testcafe/documentation/test-api/authentication/user-roles.html

Reply:
did now know it. implementing.